crates/sui/src/sui_commands.rs

```diff
@@ -994,14 +1021,20 @@ async fn genesis(
     if let Some(epoch_duration_ms) = epoch_duration_ms {
         genesis_conf.parameters.epoch_duration_ms = epoch_duration_ms;
     }
+    let committee_size = match committee_size {
+        Some(x) => NonZeroUsize::new(x),
+        None => NonZeroUsize::new(DEFAULT_NUMBER_OF_AUTHORITIES),
+    }
+    .ok_or_else(|| anyhow!("Committee size must be at least 1."))?;
```

Comment:
```suggestion
    let committee_size = match committee_size {
        Some(x) => {
          if network_config.is_some() { println!("WARNING...") }
          NonZeroUsize::new(x),
        }
        None => NonZeroUsize::new(DEFAULT_NUMBER_OF_AUTHORITIES),
    }
    .ok_or_else(|| anyhow!("Committee size must be at least 1."))?;
```
I'm guessing you would see this behavior in the `--help` given the `///` comments? but it also might not hurt to just warn/error. It's debatable though.

Reply:
That's probably a good idea to warn, though I think this needs to be done on `start` command rather than genesis.